mcs/class/System.Private.CoreLib/System.Collections.Generic/EqualityComparer.cs

```diff
@@ -48,6 +48,7 @@ static EqualityComparer<T> CreateComparer ()
 			return new ObjectEqualityComparer<T>();
 		}
 
+		[MonoTODO ("Add specialized versions")]
```

Comment:
Don't use MonoTODO, just use normal comment

Reply:
Will use // MONOTODO: in the future, so it can be searched for.